### DISCUSSION TITLE
账号被误封

### DISCUSSION THREAD

**Discussion by ssfun** [BurnHair/docs#124] [技术支持]
用户150已被封禁：重复注册 会话重复 (request id: 20240626021739299894955nYEvJ4ur)

我是挂在 one-api 和一些 Mac client 里使用、无重复注册，请帮忙核查

**Comment by BennyThink** [comment]
你是不是和朋友共享 你的bearer token，然后挂到同一个one-api去用？ 仅此一次，请勿共享 API Key

**Reply by ssfun** [reply]
谢谢，我 one-api 做了备份，挂在 render 和 huggingface space 上。是不是因为这个被误以为共享API了？

**Reply by BennyThink** [reply]
有可能，尽量不要这样做 :-)
